recipes/trf/meta.yaml

```diff
@@ -0,0 +1,19 @@
+package:
+  name: trf
+  version: "4.07b"
+
+source:
+  fn: trf407b
+  url: https://tandem.bu.edu/trf/downloads/trf407b.linux64 # [linux]
+  url: https://tandem.bu.edu/trf/downloads/trf407b.macos64 # [osx]
```

Comment:
In general we recommend to compile from source, as you never know if these binaries are relocatable and will work on old Linux machines.
But if this is to complicated I'm fine with this, I have seen this before.